@here/harp.gl/package.json

```diff
@@ -12,10 +12,13 @@
         "harpgl"
     ],
     "main": "dist/harp.js",
+    "types": "dist/harp.gl.d.ts",
```

Comment:
```suggestion
    "types": "dist/harp.d.ts",
```

Reply:
as mentioned, the type file should be named the same as the file it is covering, otherwise VSCode doesn't know that they belong together